pkg/backend/httpstate/backend.go

```diff
@@ -1278,6 +1279,23 @@ func (b *cloudBackend) createAndStartUpdate(
 		logging.V(7).Infof("Stack %s being updated to version %d", stackRef, version)
 	}
 
+	// Check if the user's org (stack's owner) has Copilot enabled. If not, we don't show the link to Copilot.
+	isCopilotEnabled, err := b.Client().IsCopilotEnabledForOrg(ctx, stackID.Owner)
```

Comment:
This is adding an additional request to the service on the start of each update. Which may be fine, but could impact performance.

Instead, have we considered returning an optional bool field from the result of `CreateUpdate` that indicates whether copilot is enabled? Are there any downsides to doing it that way, aside from adding a copilot-related field to `UpdateProgramResponse` & `CreateUpdateDetails`?

Reply:
Good idea, I added new payload into `CreateUpdateDetails` and `UpdateProgramResponse` and it looks not too ugly.